wayback-webapp/src/main/webapp/WEB-INF/query/BubbleCalendar.jsp

```diff
@@ -36,12 +36,11 @@ if(graphJspPrefix == null) {
 // graph size "constants": These are currently baked-in to the JS logic...
 int imgWidth = 0;
 int imgHeight = 75;
-int yearWidth = 49;
+int yearWidth = 60;
 int monthWidth = 5;
 int startYear = Timestamp.getStartYear();
 
-for (int year = startYear; year <= Calendar.getInstance().get(Calendar.YEAR); year++)
-    imgWidth += yearWidth;
+imgWidth = yearWidth * ( Calendar.getInstance().get(Calendar.YEAR) - startYear + 1 );
 
```

Comment:
Strictly a style comment, but in order to maintain consistency with the pre-existing code in this file, it makes sense to me to remove the extra whitespace after the open parentheses and before the close parentheses and in other changes made in this PR that add extra whitespace inside parentheses and curly braces.